src/config.ts

```diff
@@ -34,17 +35,16 @@ const enableMultiTenancy = ENABLE_MULTI_TENANCY === 'true';
 // https://github.com/serverless/serverless/issues/7087
 // As of May 14, 2020, this bug has not been fixed and merged in
 // https://github.com/serverless/serverless/pull/7147
-const issuerEndpoint =
-    process.env.ISSUER_ENDPOINT === '[object Object]' || process.env.ISSUER_ENDPOINT === undefined
-        ? 'https://OAUTH2.com'
+const defaultEndpoint = 'https://OAUTH2.com';
+let issuerEndpoint =
+    process.env.ISSUER_ENDPOINT === '[object Object]' ||
+    process.env.ISSUER_ENDPOINT === undefined ||
+    process.env.ISSUER_ENDPOINT === ''
+        ? defaultEndpoint
```

Comment:
This code format change is probably fine. But if we can avoid making code formatting changes, the easier it should be to merge in changes to our fork. Thanks.

Reply:
The code format must be done by `yarn lint-fix` automatically, otherwise the deployment would fail.

Reply:
But isn't the deployment succeeding without the change?

Reply:
I added `process.env.ISSUER_ENDPOINT === ''`, and it changed the format.

Reply:
I decided to remove process.env.ISSUER_ENDPOINT === ''